docs/snippets/web-components/msw-addon-configure-handlers-http.js.mdx

```diff
@@ -34,21 +34,28 @@ const TestData = {
 
 export const MockedSuccess = {
   parameters: {
-    msw: [
-      rest.get('https://your-restful-endpoint/', (_req, res, ctx) => {
-        return res(ctx.json(TestData));
-      }),
-    ],
+    msw: {
+      handlers: [
+        http.get('https://your-restful-endpoint/', () => {
+          return new HttpResponse.json(TestData);
+        }),
+      ],
+    },
   },
 };
 
 export const MockedError = {
   parameters: {
-    msw: [
-      rest.get('https://your-restful-endpoint', (_req, res, ctx) => {
-        return res(ctx.delay(800), ctx.status(403));
-      }),
-    ],
+    msw: {
+      handlers: [
+        http.get('https://your-restful-endpoint', async () => {
+          await delay(800);
```

Comment:
Don't forget to add `http`, `HttpResponse` and `delay` to the msw imports at line 4, also remove `rest`